### PR TITLE
DOCS: Libraries for Local Distribution image recreation for master

### PR DESCRIPTION
**Pull request by msmykx-intel**
Recreation of image in svg format for Libraries for Local Distribution article.

### CODE REVIEW

File: docs/OV_Runtime_UG/deployment/local-distribution.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ The `plugins.xml` file with information about inference devices must also be tak
 
 The picture below presents dependencies between the OpenVINO Runtime core and pluggable libraries:
 
-![deployment_full]
+![](../../img/deployment_full.svg)
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/deployment/local-distribution.md:23]: can we use more traditional color for the picture?

**Review by sgolebiewski-intel — APPROVED** [review]

File: docs/OV_Runtime_UG/deployment/local-distribution.md (hunk 1 above)
Comment (msmykx-intel) [reply docs/OV_Runtime_UG/deployment/local-distribution.md:23]: Hi :) I used colors from brand color guideline to fit with colors of the original picture. If You suggest to change them, then which colors will fit the most?

Comment (ilya-lavrenov) [reply docs/OV_Runtime_UG/deployment/local-distribution.md:23]: Color like here https://docs.openvino.ai/latest/ovms_what_is_openvino_model_server.html

Comment (msmykx-intel) [reply docs/OV_Runtime_UG/deployment/local-distribution.md:23]: I committed the new color version. I have to use colors from brand color guideline, so these are the closest to the ones from what You linked.

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**